configs/.sass-lint.yml

```diff
@@ -5,7 +5,7 @@ rules:
   placeholder-in-extend: 0
 
   # Mixins
-  mixins-before-declarations: 1
+  mixins-before-declarations: 0
```

Comment:
This doesn't work too good with the media query mixins.

Meant lots of changes for the repo I was putting this into so once we agree a standard we can put it back in.